<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Encoder</class>
 <widget class="QWidget" name="Encoder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>559</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>397</width>
    <height>559</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>396</width>
       <height>549</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>395</width>
        <height>557</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>394</width>
        <height>547</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>377</width>
           <height>184</height>
          </size>
         </property>
         <property name="title">
          <string>Cast Encoder</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="lame">
            <property name="text">
             <string>LAME (MP3)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="aac">
            <property name="text">
             <string>AAC (FAAC)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="customr">
            <property name="text">
             <string>Custom:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="custom">
            <property name="text">
             <string>lame -r -s 44100 -b %i -</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>377</width>
           <height>319</height>
          </size>
         </property>
         <property name="title">
          <string>Cast Config</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Mountpoint/Server:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="server">
            <property name="text">
             <string>listen.superb-radio:8000/mount</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pass">
            <property name="text">
             <string>password</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::PasswordEchoOnEdit</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Station Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="name">
            <property name="text">
             <string>Superb Radio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Station URL:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="url">
            <property name="text">
             <string>http://example.org</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Genre:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="genre">
            <property name="text">
             <string>Generic</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="descr">
            <property name="text">
             <string>Superb Station</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Quality (Kb/sec):</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QSlider" name="quas">
              <property name="minimum">
               <number>8</number>
              </property>
              <property name="maximum">
               <number>320</number>
              </property>
              <property name="singleStep">
               <number>8</number>
              </property>
              <property name="value">
               <number>128</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::NoTicks</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="qua">
              <property name="text">
               <string>128</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StartEncoder">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StopEncoder">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
